### PR TITLE
PMM-6535 Rule to add graph to explore url

### PR DESCRIPTION
**Pull request by tiagomotasantos**
Adds missing "graph" to the URL when opening explore in new tab.

https://github.com/Percona-Lab/pmm-submodules/pull/1103

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -94,6 +94,7 @@
     # Grafana
     rewrite ^/$ $scheme://$http_host/graph/;
     rewrite ^/graph$ /graph/;
+    rewrite ^/explore(.*)$ /graph/explore$1 permanent;
```
Comment (adivinho) [inline nginx.conf:97]: Could you move the line into the section below?
Screenshot_20200914_183811

**Review by adivinho — CHANGES_REQUESTED** [review]

File: nginx.conf (hunk 1 above)
Comment (tiagomotasantos) [reply nginx.conf:97]: Yes I can, but then the comment will mislead saying that this line can be removed after fixing that problem, which is not the case for this. Should I still move it?

Comment (AlekSi) [reply nginx.conf:97]: Do we have upstream issue for that?

Comment (tiagomotasantos) [reply nginx.conf:97]: No. But I don't think the problem is upstream since they don't use /graph.

Comment (AlekSi) [reply nginx.conf:97]: They provide the root_url parameter, and they should honor it. Please check the linked issue, then find or create n issue for this location, then add a link there.

Comment (tiagomotasantos) [reply nginx.conf]: You are right. Didn't find an issue so created one [here](https://github.com/grafana/grafana/issues/27588) and added the link in a comment.

File: nginx.conf [hunk 2]
```diff
@@ -105,6 +105,9 @@
     rewrite ^/(user/password/send-reset-email)$ $scheme://$http_host/graph/$1;
     rewrite ^/(login)$ $scheme://$http_host/graph/$1; 
 
+    # Remove next line if bug is fixed https://github.com/grafana/grafana/issues/27588
+    rewrite ^/explore(.*)$ /graph/explore$1 permanent;
```
Comment (AlekSi) [inline nginx.conf:109]: It should not be a permanent redirect if we intend to fix in the future.
Please also use the same style as above with $scheme, etc

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: nginx.conf (hunk 2 above)
Comment (tiagomotasantos) [reply nginx.conf]: Done!

**Review by AlekSi — APPROVED** [review]

**Review by adivinho — APPROVED** [review]

**Pull request merged**